cmd/cosign/cli/options/clean.go

```diff
@@ -27,5 +27,6 @@ var _ Interface = (*CleanOptions)(nil)
 func (c *CleanOptions) AddFlags(cmd *cobra.Command) {
 	c.Registry.AddFlags(cmd)
 	cmd.Flags().StringVarP(&c.CleanType, "type", "", "all", "a type of clean: <signature|attestation|sbom|all> (default: all)")
+	// TODO: Rename to --skip-confirmation for consistency?
```

Comment:
it makes sense a lot to me, may I open an issue for this? @imjasonh

Reply:
Sure! I think we'll probably want `--force` to continue to exist for 1+ release, with a warning that it's renamed to `--skip-confirmation`, and use `--force`'s value to set `o.SkipConfirmation`. If both are set and they disagree, then fail. Does that sound right?

Reply:
Absolutely !

Reply:
Issue: https://github.com/sigstore/cosign/issues/2044